.travis.yml

```diff
@@ -2,9 +2,6 @@ dist: xenial
 language: python
 
 python:
-    - "3.4"
-    - "3.5"
-    - "3.6"
     - "3.7"
     - "3.8"
     - "pypy3.6-7.1.1"
```

Comment:
Could you please add back Python 3.6?

The newest version for pypy3  is.... pypy3 :-)

See https://docs.travis-ci.com/user/languages/python/#pypy-support

Reply:
Sure thing !